packages/idempotency/tests/helpers/populateEnvironmentVariables.ts

```diff
@@ -1,7 +1,7 @@
 // Reserved variables
 process.env._X_AMZN_TRACE_ID = '1-abcdef12-3456abcdef123456abcdef12';
 process.env.AWS_LAMBDA_FUNCTION_NAME = 'my-lambda-function';
-process.env.AWS_EXECUTION_ENV = 'nodejs16.x';
+process.env.AWS_EXECUTION_ENV = 'nodejs20.x';
```

Comment:
should we put the default to lowest nodejst version as in other files?

Reply:
Let's discuss it in [this thread](https://github.com/aws-powertools/powertools-lambda-typescript/pull/2717#discussion_r1662133826) and settle both.